test/src/org/labkey/test/tests/idri/FormulationsTest.java

```diff
@@ -390,29 +385,21 @@ protected void defineVisualInspectionAssay()
 
         log("Defining Visual Inspection Assay");
         clickAndWait(Locator.linkWithText("Manage Assays"));
-        clickButton("New Assay Design");
+        ReactAssayDesignerPage assayDesignerPage = _assayHelper.createAssayDesign("Visual Inspection", VIS_INSPEC_ASSAY)
+            .setDescription(VIS_INSPEC_ASSAY_DESC);
 
-        assertTextPresent("Visual Formulation Time-Point Data");
-        checkCheckbox(Locator.radioButtonByNameAndValue("providerName", "Visual Inspection"));
-        clickButton("Next");
+        assayDesignerPage.goToBatchFields();
+        assertElementPresent(Locator.tagWithClass("div", "domain-form-no-field-panel"));
 
-        waitForElement(Locator.xpath("//input[@id='AssayDesignerName']"), WAIT_FOR_JAVASCRIPT);
-        setFormElement(Locator.xpath("//input[@id='AssayDesignerName']"), VIS_INSPEC_ASSAY);
-        setFormElement(Locator.xpath("//textarea[@id='AssayDesignerDescription']"), VIS_INSPEC_ASSAY_DESC);
-        fireEvent(Locator.xpath("//input[@id='AssayDesignerName']"), SeleniumEvent.blur);
+        DomainFormPanel runPanel = assayDesignerPage.goToRunFields();
+        assertThat("Missing expected run property", runPanel.getField("LotNumber"), CoreMatchers.notNullValue());
 
-        assertTextPresent(
-                // Batch Properties
-                "No fields have been defined.",
-                // Run Properties
-                "LotNumber",
-                // Result Properties
-                "Pass",
-                "Color",
-                "Phase");
+        DomainFormPanel resultPanel = assayDesignerPage.goToFieldProperties("Result Properties");
+        assertThat("Missing expected result property", resultPanel.getField("Pass"), CoreMatchers.notNullValue());
+        assertThat("Missing expected result property", resultPanel.getField("Color"), CoreMatchers.notNullValue());
+        assertThat("Missing expected result property", resultPanel.getField("Phase"), CoreMatchers.notNullValue());
```

Comment:
Verify fields in bulk. See comment for `defineParticleSizeAssay`.

Reply:
Done in new fb.